app/signals/apps/feedback/rest_framework/views.py

```diff
@@ -1,51 +1,62 @@
 # SPDX-License-Identifier: MPL-2.0
-# Copyright (C) 2019 - 2022 Gemeente Amsterdam
+# Copyright (C) 2019 - 2023 Gemeente Amsterdam
 """
 Views for feedback handling.
 """
-from datapunt_api.pagination import HALPagination
-from rest_framework import mixins, viewsets
+from typing import Union
+
+from django.http import Http404
+from drf_spectacular.utils import extend_schema, extend_schema_view
+from rest_framework.mixins import ListModelMixin, RetrieveModelMixin, UpdateModelMixin
+from rest_framework.status import HTTP_200_OK, HTTP_404_NOT_FOUND, HTTP_410_GONE
+from rest_framework.viewsets import GenericViewSet
 
 from signals.apps.feedback.models import Feedback, StandardAnswer
 from signals.apps.feedback.rest_framework.exceptions import Gone
 from signals.apps.feedback.rest_framework.serializers import (
     FeedbackSerializer,
     StandardAnswerSerializer
 )
+from signals.schema import GenericErrorSerializer
 
 
-class StandardAnswerViewSet(viewsets.GenericViewSet, mixins.ListModelMixin):
+class StandardAnswerViewSet(ListModelMixin, GenericViewSet):
     """View to list all currently visible Standard Answers."""
     serializer_class = StandardAnswerSerializer
-    queryset = StandardAnswer.objects.select_related('topic')\
-        .filter(is_visible=True).order_by('topic__order', 'order', '-id')
-    pagination_class = HALPagination
-
-
-class FeedbackViewSet(
-        viewsets.GenericViewSet, mixins.UpdateModelMixin, mixins.RetrieveModelMixin):
+    queryset = StandardAnswer.objects.select_related(
+        'topic'
+    ).filter(
+        is_visible=True
+    ).order_by(
+        'topic__order',
+        'order',
+        '-id',
+    )
+
+
+@extend_schema_view(
+    retrieve=extend_schema(responses={HTTP_200_OK: FeedbackSerializer,
+                                      HTTP_404_NOT_FOUND: GenericErrorSerializer,
+                                      HTTP_410_GONE: GenericErrorSerializer}),
+    update=extend_schema(responses={HTTP_200_OK: FeedbackSerializer,
+                                    HTTP_404_NOT_FOUND: GenericErrorSerializer,
+                                    HTTP_410_GONE: GenericErrorSerializer}),
+    partial_update=extend_schema(responses={HTTP_200_OK: FeedbackSerializer,
+                                            HTTP_404_NOT_FOUND: GenericErrorSerializer,
+                                            HTTP_410_GONE: GenericErrorSerializer}),
+)
+class FeedbackViewSet(UpdateModelMixin, RetrieveModelMixin, GenericViewSet):
     """View to receive complaint/client feedback."""
     serializer_class = FeedbackSerializer
     queryset = Feedback.objects.all()
 
-    def _raise_if_too_late_or_filled_out(self):
-        """Raise HTTP 410 if feedback sent too late or twice or more."""
-        obj = self.get_object()
+    def get_object(self) -> Union[Feedback, Http404, Gone]:
```

Comment:
Union is no longer necessary, you can also define it as `Feedback | Http404 | Gone`